docs/Эталоны.md

```
@@ -0,0 +1,87 @@
+# Эталоны
+
+Для обозначения эталонного метода бенчмарка используйте аннотацию `&Эталон`.
+Это позволяет сравнивать производительность других методов с выбранным эталоном.
+
+### API
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)